sdk/ml/azure-ai-ml/azure/ai/ml/_arm_deployments/arm_templates/workspace_base.json

```diff
@@ -437,32 +437,51 @@
                 "description": "Feature store online store config"
             }
         },
-        "offline_store_target": {
+        "offlineStoreStorageAccountOption": {
             "type": "string",
-            "defaultValue": "",
+            "defaultValue": "existing",
+            "allowedValues": [
+                "new",
+                "existing",
+                "none"
```

Comment:
when to use "none"?

Reply:
Normal workspace should use none, I will update the default to none.